addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3533,12 +3865,24 @@ var SnippetsMenu = Widget.extend({
      */
     _onUndo: async function () {
         this.options.wysiwyg.undo();
+        // Resizing all the grids.
+        // TODO maybe to remove when history will be fixed.
+        const $gridModeRows = this.getEditableArea().find('.row.o_grid_mode');
+        for (const rowEl of $gridModeRows) {
+            gridUtils.resizeGrid(rowEl);
+        }
```

Comment:
A bug you reported somewhere? Editor team?

Reply:
I made a PR (I talked about it with you some time ago) and I created the task yesterday. I was going to ask you to "validate" it if it's ok with you. (https://www.odoo.com/web#id=2968749&menu_id=4720&cids=1&action=333&active_id=974&model=project.task&view_type=form)

Reply:
I remember, just did not know where the task was :+1: I moved the task to RD Dev since you already made a PR, once it's ready, follow the usual process then make it green, we'll merge it later :+1: